lib/path/index.d.ts

```diff
@@ -1,3 +1,10 @@
+/**
+ * Gets the prefix of an absolute Windows, UNC, or Unix path or a URL that indicates it's absolute.
+ * @param {string} path Path to test
+ * @returns {string|null} the prefix, or null if path is relative
+ */
+export function absolutePrefix(path: string): (string|null);
+
```

Comment:
Is there a script to update these `index.d.ts` files? I saw that `npm run build:types` updates the one at the root, but I didn't see anything similar for these, so I just made the change manually.